tests/snapshots/assets__check_file@cetz-tree.typ-120.snap

```diff
@@ -176,15 +174,9 @@ input_file: tests/assets/cetz-tree.typ
   let node-position(node) = {
     if direction == "bottom" {
       return (node.x, -node.y)
-    } else if direction == "top" {
-      return (node.x, node.y)
-    } else if direction == "left" {
+    } else if direction == "top" { return (node.x, node.y) } else if direction == "left" {
       return (-node.y, node.x)
-    } else if direction == "right" {
-      return (node.y, node.x)
-    } else {
-      panic(message: "Invalid tree direction.")
-    }
```

Comment:
looks like currently `else-if` pattern is not well recognized. They should be stmt-like-if but not expr-like-if